package.json

```diff
@@ -30,6 +30,7 @@
     "generate:docs": "npx compodoc -p tsconfig.compodoc.json && ./scripts/zip-docs.sh",
     "generate:translations": "ts-node ./scripts/generate-translations",
     "lint": "ng lint",
+    "lint-fix": "ng lint --fix",
```

Comment:
We already have too many scripts. A flag can be just passed after the command. No need to add another script, just for this case.